portfolio/src/main/webapp/script.js

```diff
@@ -15,9 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
+const contactLiteral = "contact";
+const nameLiteral = "Name";
+const emailLiteral = "Email";
+const numberLiteral = "Number";
+const deleteLiteral = "Delete";
```

Comment:
These are not the most descriptive, and as such not that useful to the reader. Try to think about how the literals will be used, or what they identify. 

https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c

Reply:
should hopefully be good now